src/components/Home/index.tsx

```diff
@@ -78,43 +82,29 @@ export default function Home({ recentPosts }: BlogProps) {
-              <p>Nearly all contributions are welcome, for anything from the main flight control firmware, to the configurator, as well as this site</p>
+              <p>Betaflight is 'Open Source' code, so you can view and contribute to the project on GitHub.</p>
+              <p>The team has a robust review system in order to maintain clean code, and we are always looking for talented contributors.</p>
             </AboutCard>
             <AboutCard className="border-blue-500 text-blue-500" title="OSD" Icon={CameraIcon}>
-              <p>
-                With the Betaflight OSD you can get all relevant flight metrics directly into your FPV video feed. An easy to use drag-and-drop configuration allows the placement of values like used
-                mAh and LiPo Voltage readings.
-              </p>
-              <p>Additionally you can change most firmware settings using stick commands without even removing your goggles.</p>
+              <p>With the Betaflight On Screen Display you can use drag-and-drop to set up key flight metrics into your FPV video feed. </p>
+              <p> This allows data such as battery metrics, speed, altitude and home direction.</p>
             </AboutCard>
             <AboutCard className="border-orange-600 text-orange-600" title="Safety Features" Icon={ShieldCheckIcon}>
-              <p>
-                Drones are extremely dangerous toys. As incidents with suddenly spinning props often end with serious injury, Betaflight implements several safety features to prevent dangerous
-                situations occurring.
-              </p>
-              <p>Features are implemented to prevent arming when the drone isn't leveled, including disabling motors in case of wrong motor or incorrect flight controller orientation.</p>
+              <p>Alerts for and arming blocks for improper setup, and disarm mechanisms are built in to avoid accidents. </p>
+              <p>A comprehensive failsafe mechanism is featured to assist in flight issues</p>
             </AboutCard>
             <AboutCard className="border-purple-600 text-purple-600" title="Flight Dynamics" Icon={JetIcon}>
-              <p>
-                Betaflight not only strives to put the best flight code on your controller, it also looks to satisfy such requirements as perfect performance, tiny footprint, full feature set and a
-                strong quality assurance process.
-              </p>
-              <p>
-                To assist in this endeavour all flight related data can be logged into a blackbox for later analysis. So pilots and technicians can objectively base their tune on the most appropriate
-                data.
-              </p>
+              <p>Betaflight was created for cutting edge flight performance. This has been achieved optimising reaction time to disturbances, and increasing the accuracy of stick tracking.</p>
```

Comment:
[...] been achieved \*by* optimi\*z*Ing [...]

Reply:
...well I'm english and we invented the language ;)

Reply:
S/Z was just my keyboard correcting me, I'm honestly lost on the spelling there between UK/US, sorry 😅

Reply:
No probs, it can be confusion. I think considering our UK contingent we can let this version stand ;)